anti-behavioral-analysis/debugger-detection.md

```diff
@@ -82,11 +82,14 @@ Details on methods of detecting debuggers are given in the references; many are
 
 |Name|Date|Method|Description|
 |---|---|---|---|
-|[**Redhip**](../xample-malware/rebhip.md)|January 2011|--|Redhip uses general approaches to detecting user level debuggers (e.g., Process Environment Block 'Being Debugged' field), as well as specific checks for kernel level debuggers like SOFTICE. [[4]](#4)|
-|[**Gamut**](../xample-malware/gamut.md)|2014|--|The malware detects debuggers using an INT 03h trap and IsDebuggerPresent[[8]](#8)|
-|[**Rombertik**](../xample-malware/rombertik.md)|2015|--|an anti-analysis function within the packer is called to check the username and filename of the executing process for strings like “malwar”, “sampl”, “viru”, and “sandb”. [[9]](#9)|
-|[**Poison-Ivy**](../xample-malware/poison-ivy.md)|2005|--|Poison Ivy Variant checks for breakpoints and exits immediately if found [[10]](#10)|
-
+|[**Redhip**](../xample-malware/rebhip.md)|January 2011|B0001, B0001.035, B0001.032|Please see the Redhip malware page for details. [[4]](#4)|
```

Comment:
There should only be one method per row where the description matches the content from the malware page. In this case, there will be three rows for Redhip. 

Also, where a row corresponds to the behavior and not more specifically to a method, the Method column should be empty. But the description as it applies to the behavior should be given.